frontend/templates/product-list/ProductListView.tsx

```diff
@@ -63,11 +62,7 @@ export function ProductListView({
          />
          <MetaTags productList={currentProductList} />
          <SecondaryNavigation productList={productList} />
-         <VStack
-            align="stretch"
-            spacing={{ base: 4, md: 6 }}
-            py={{ base: 4, md: 6 }}
-         >
```

Comment:
0e29d6cdf955934852b64e2786df4b2535a99138 might be a design pattern I'm not aware of - would you mind linking me or briefly explaining the motivation for making child components responsible for their margins?

I would naively assume defining the same margin on a list of similar components in the parent is more maintainable.

https://mxstbr.com/thoughts/margin/

Reply:
> https://mxstbr.com/thoughts/margin/

I remember having read this article by Max, definitely a good read 👍 
As much as I agree on the thesis and I try to adhere to that concept, I still think that sometimes breaking the rules  [is more convenient](https://dev.37signals.com/globals-callbacks-and-other-sacrileges/).

Here we're doing this out of necessity because now we have new sections in the product list page that don't want to have a margin between them, so it's not possible to have a `VStack` with uniform margins.  

<img width="1728" alt="Screenshot 2023-08-23 at 16 37 42" src="https://github.com/iFixit/react-commerce/assets/4640135/265fe4cf-9469-4063-8f2a-419c85f108df">

Reply:
> now we have new sections in the product list page that don't want to have a margin between them

I didn't notice this, thanks for point it out.

I agree that absolutism can hold us back, thanks for sharing the article 💯